catalog/spatial/csw/spatial-csw-endpoint/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/endpoint/mappings/CswRecordMapperFilterVisitor.java

```diff
@@ -272,6 +295,19 @@ public Object visit(PropertyIsBetween filter, Object extraData) {
 
     @Override
     public Object visit(PropertyIsEqualTo filter, Object extraData) {
+        FilterFactory2 factory = getFactory(extraData);
+
+        if (filter.getExpression1() instanceof Function) {
```

Comment:
❓ Is this introducing tech debt we'll want to pay down, then? Have you created tickets to address that?